packages/babel-plugin-transform-block-scoping/test/fixtures/general/hoisting/output.js

```diff
@@ -10,29 +10,19 @@ var _loop = function (i) {
   });
 };
 
-var _iteratorNormalCompletion = true;
-var _didIteratorError = false;
-var _iteratorError = undefined;
+var _iteratorHelper = babelHelpers.createForOfIteratorHelper(nums),
+    _step;
 
 try {
-  for (var _iterator = nums[Symbol.iterator](), _step; !(_iteratorNormalCompletion = (_step = _iterator.next()).done); _iteratorNormalCompletion = true) {
+  for (_iteratorHelper.s(); !(_step = _iteratorHelper.n()).done;) {
     var i = _step.value;
     var x;
     var f;
```

Comment:
This is a bug unrelated to this PR: `f` and `x` should be declared in the `_loop` function.

Reply:
I think they have to stay here. If we moved the `var` inside the `_loop` function, they would no longer declare the variable in the function scope (it's not a block `let`/`const`).